level1c4pps/vgac2pps_lib.py

```diff
@@ -255,8 +462,9 @@ def split_scene_at_midnight(scene):
 
 
 def process_one_scene(scene_files, out_path, engine='h5netcdf',
-                      all_channels=False, pps_channels=False, orbit_n=0, as_noaa19=False, avhrr_channels=False,
-                      split_files_at_midnight=True):
+                      all_channels=False, pps_channels=False, orbit_n=0,
+                      as_noaa19=False, avhrr_channels=False,
+                      split_files_at_midnight=True, SBAF_version='v3'):
```

Comment:
Assuming we are referring to 'SBAF_version', I've renamed it to lowercase and set the default value to False

Reply:
see comments on the command line interface. It would be cleaner to combine as_noaa19 and sbaf_version to one argument, maybe name `noaa19_sbaf_version` with `None` as default.  And the change the code below to `if noaa19_sbaf_version is not None`